collections/_article/dev-snapshot-godot-4-2-dev-3.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 3"
+excerpt: "One month into the release cycle for Godot 4.2, things are well under way. This third dev snapshot features a lot of improvements to C# and rendering."
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-3.webp
+image_caption_title: ""
+image_caption_description: ""
+date: 2023-08-11 13:00:00
+---
+
+Development is well under way for our next milestone, Godot 4.2. After our [dev 2 snapshot](/article/dev-snapshot-godot-4-2-dev-2/), we kept merging a number of new features and important bug fixes, so we're ready for a third dev snapshot to put those under broader user testing. It's been around a month since we started this release cycle, with roughly two months to go for 4.2 beta 1, and a fourth month to reach stable in early November 2023.
```

Comment:
```suggestion
Development is well under way for our next milestone, Godot 4.2. After our [dev 2 snapshot](/article/dev-snapshot-godot-4-2-dev-2/), we kept merging new features and important bug fixes, and gathered quite a number of them — so now we're ready for the third dev snapshot to put those under broader user testing. It's been around a month since we started this release cycle, with roughly two months to go until 4.2 beta 1, and then another month to reach stable in early November 2023.
```